back/app.py

```diff
@@ -4,7 +4,7 @@
 pyinstaller package to create a standalone executable.
 """
 
-import multiprocessing
+import multiprocessing, asyncpg
```

Comment:
It would be great to avoid this dependency for non-postgresql users. We could add a specific field to whombat config or use environment variables to check if postgresql will be used.